js/bruh-dash.js

```diff
@@ -1,83 +1,297 @@
 var global = window || GLOBAL;
 
+
 global.bruhdash = {
-  chunk: function(){
 
+  chunks:chunks,
+
+  chunk: function(array, number){
+    if(isNaN(number))
```

Comment:
For this one, we'd like to have `chunk` to set a default value of `1` for `number.

Expected output:
```javascript
bruhdash.chunk(['a', 'b', 'c', 'd'])

[["a"],["b"],["c"],["d"]]
```